api/src/LinkyLink/Helpers/UserAuth.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using LinkyLink.Models;
 using Microsoft.AspNetCore.Http;
+using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 
 namespace LinkyLink.Helpers
 {
     /// <summary>
     /// This class provides the method to read the identity of the authenticated user and extracts identity provider and email address.
     /// </summary>
+    [ExcludeFromCodeCoverage]
     public class UserAuth
```

Comment:
This class is tested as part of Integration test and its difficult to test HttpContextAccessor in isolation thats why I excluded it from code coverage.

Reply:
we should have this comment 

> This class is tested as part of Integration test and its difficult to test HttpContextAccessor in isolation thats why I excluded it from code coverage.

this should be a comment in code too so we know later when/if we figure out a good way to test

https://css-tricks.com/the-art-of-comments/